docs/open-source/falcon-client/code-splitting.md

```diff
@@ -28,7 +28,7 @@ export default () => (
 )
 ```
 
-When you create your project using [`create-falcon-app`](getting-started/installation#create-falcon-app-cli-tool) Falcon provides a wrapper function for `loadable`. This wrapper function shows a spinner while the component is loading to indicate to the user that the contents of the page are currently being fetched. Using this wrapper function is optional, but it is recommended to provide a fallback state to `loadable` if you choose to write your own.
+When you create your project using [`create-falcon-app`](/docs/open-source/getting-started/installation#create-falcon-app-cli-tool) Falcon provides a wrapper function for `loadable`. This wrapper function shows a spinner while the component is loading to indicate to the user that the contents of the page are currently being fetched. Using this wrapper function is optional, but it is recommended to provide a fallback state to `loadable` if you choose to write your own.
```

Comment:
totally unrelated to your pwa problem but did doc linking break?

Reply:
> totally unrelated to your pwa problem but did doc linking break?

It didn't break but relative links were flagged by the broken link checker.

Reply:
I see, will try to not report this.

I think if you reference the md file (with extension) it wouldn't be flagged

Reply:
That's good to know. Thanks

Reply:
FYI this broken link will be fixed in next version so you can revert to relative links if you want

Reply:
Thanks Slorber. That's great to know

Reply:
that should be fixed in alpha 60. Would be interested to know if you can safely revert those changes